server/mailet/droplists-cassandra/src/main/java/org/apache/james/droplists/cassandra/CassandraDropListsModule.java

```diff
@@ -0,0 +1,36 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.droplists.cassandra;
+
+import org.apache.james.droplists.api.DropList;
+import org.apache.james.webadmin.routes.DropListRoutes;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Scopes;
+
+public class CassandraDropListsModule extends AbstractModule {
+    @Override
+    protected void configure() {
+        bind(CassandraDropList.class).in(Scopes.SINGLETON);
+        bind(DropList.class).to(CassandraDropList.class);
+
+        bind(DropListRoutes.class).in(Scopes.SINGLETON);
```

Comment:
Why binding the routes in Singleton?

Reply:
I should have used Multibinder...

Reply:
You can not use parent multi-binder in a child module...

This could be doable if we use module combinaison for loading extensions but this leads to a lower isolation between components. Child modules were historically choosen on the project to provide higher level isolation...

Anyway https://github.com/apache/james-project/pull/2256#discussion_r1612285222 will address this.